supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -0,0 +1,26 @@
+[[support]]
+= Support
+
+[discrete]
+== Technology Preview
+
+Technology Preview documentation guidelines apply to any form of early access feature that is not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.
+
+* Add an admonition labeled IMPORTANT at the beginning of the Technology Preview content.
+* Use initial uppercase capitalization, that is, Technology Preview.
+* Include a brief description of the Technology Preview feature in the Release Notes.
+* Maintain a list of features that are currently in Technology Preview status in the Release Notes.
+
+When a minor release becomes generally available, remove the IMPORTANT admonition from the Release Notes and any other document that includes content about the feature.
+
+Use the following template. Replace _<feature_name>_ with the feature name:
+
+
+[IMPORTANT]
+====
+_<feature_name>_ is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+====
+
+See link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope] for more information about the support scope of Red Hat Technology Preview features.
```

Comment:
Can you move this statement inside the IMPORTANT note? It's part of what people should actually be including in their docs.

Though actually when I look at it again, I believe the exact wording from the template is:

```suggestion
For more information about the support scope of Red Hat Technology Preview
features, see https://access.redhat.com/support/offerings/techpreview/.
```